lib/apress/utils/engine.rb

```diff
@@ -15,12 +15,8 @@ class Engine < ::Rails::Engine
 
       require cd + '/extensions/uri/parsers_overrides'
 
-      if RUBY_VERSION < '2.2'
-        URI::Parser.send(:include, ::Apress::Utils::Extensions::Uri::ParsersOverrides)
-      else
-        URI::RFC2396_Parser.send(:include, ::Apress::Utils::Extensions::Uri::ParsersOverrides)
-        URI::RFC3986_Parser.send(:include, ::Apress::Utils::Extensions::Uri::ParsersOverrides)
-      end
+      URI::RFC2396_Parser.send(:include, ::Apress::Utils::Extensions::Uri::ParsersOverrides)
```

Comment:
[Correctable] Lint/SendWithMixinArgument: Use include ::Apress::Utils::Extensions::Uri::ParsersOverrides instead of send(:include, ::Apress::Utils::Extensions::Uri::ParsersOverrides).